Update enterprise admin section (#2183)
hashicorp/terraform-website: content/enterprise/admin/infrastructure/admin.mdx

---
page_title: Infrastructure Administration - Terraform Enterprise
---

> # Terraform Enterprise Admin CLI commands

Reviewer: We always want to use title case when we have titles of any level :-) 

```suggestion
# Terraform Enterprise Admin CLI Commands
```

> The [Active/Active](/enterprise/install/automated/active-active) operational mode disables the Replicated Admin Console. Instead, it provides admin commands to change the configuration, stop the application safely, and produce support bundles. You must use SSH to log in to a node in the Active/Active cluster to run these commands from the command line.

Reviewer: Just thinking through flow - that last sentence felt a bit wordy. Okay also - should we say "Active/Active and Standalone?  It's both for the latest versions onward, right?  

```suggestion
The [Active/Active](/enterprise/install/automated/active-active) operational mode disables the Replicated Admin Console. Instead, it provides admin CLI commands to change the configuration, stop the application safely, and produce support bundles. You must use SSH to log in to a node in the Active/Active cluster to run these commands.
```

Author: For standalone, the Replicated Admin Console is not disabled so I think leaving it just like your suggested change is fine.

> The [Active/Active](/enterprise/install/automated/active-active) operational mode disables use of the Replicated Admin Console.  To accommodate this, there are now admin commands to facilitate configuration changes, safe application stops, and producing support bundles. There is a new container in the TFE architecture to support this called "tfe-admin". You must login to a node in the Active/Active cluster using SSH to run these commands from the command line. 

Reviewer: Do we need to tell folks here about the new container, or can we just tell them how to log in and run the commands? This shows my lack of knowledge of TFE - please let me know if that should be added back in. BUT I made a few edits to the rest to make it flow better!

```suggestion
The [Active/Active](/enterprise/install/automated/active-active) operational mode disables the Replicated Admin Console. Instead, it provides admin commands to change the configuration, stop the application safely, and produce support bundles. You must use SSH to log in to a node in the Active/Active cluster to run these commands from the command line. 
```
